tests/integration/024_cloud_credentials/conf.tf

```diff
@@ -0,0 +1,15 @@
+terraform {
+  backend "local" {
+  }
+  required_providers {
+    env0 = {
+      source = "terraform-registry.env0.com/env0/env0"
+    }
```

Comment:
Is this a new thing or am I wrong? I mean, shouldn't the integration tests use the local version and not the version from the public registry?

Reply:
yes. This is my bad.

Reply:
@yaronya - actually looking at it again. This is not new. All tests are like this.
I guess it's somehow working as expected...

Reply:
This is not the real registry (`terraform-registry.env0.com` doesn't exist).
This is set up in `buildFakeTerraformRegistry` in `harness.go`